assets/src/components/detours/detourMap.tsx

```diff
@@ -59,6 +59,12 @@ interface DetourMapProps {
    * Callback fired when the undo button is clicked.
    */
   onUndoLastWaypoint: () => void
+
+  /*
+   * Center and zoom to position the map correctly when it renders
+   */
+  center: LatLngLiteral
```

Comment:
I was thinking about this, and it actually does feel right to put this here, since it does govern behavior of the detour map when it renders. But I wanted to raise it anyway, since I put this here more out of convenience than anything else.